tests/rkt_image_rm_test.go

```diff
@@ -38,7 +38,45 @@ const (
 	stage1App = "coreos.com/rkt/stage1"
 )
 
-func TestImageRunRm(t *testing.T) {
+func TestImageRunRmName(t *testing.T) {
+	imageFile := patchTestACI(unreferencedACI, fmt.Sprintf("--name=%s", unreferencedApp))
+	defer os.Remove(imageFile)
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	cmd := fmt.Sprintf("%s --insecure-options=image fetch %s", ctx.Cmd(), imageFile)
+	t.Logf("Fetching %s: %v", imageFile, cmd)
```

Comment:
Printing the command is not necessary, it will be printed by `spawnAndWaitOrFail`.